MQTicketService/Dockerfile

```diff
@@ -11,13 +11,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-# Specify base image and platform for the build
-ARG imageName=icr.io/ibm-messaging/mq:latest
-ARG platName=docker.io/library/maven:3.6-openjdk-11
+# Specify default base images 
+ARG baseImageRunStage=icr.io/ibm-messaging/mq:latest
```

Comment:
User needs to know to specify this if they want to build on arm64 based on the MQ image they built. Is this included in a README?